package.json

```diff
@@ -191,9 +191,9 @@
     "webpack-cli": "^5.1.4",
     "yargs": "^17.7.2"
   },
-  "packageManager": "pnpm@9.13.2",
+  "packageManager": "pnpm@9.14.2",
   "engines": {
-    "pnpm": "9.13.2"
+    "pnpm": "9.14.2"
```

Comment:
follow the latest on master. otherwise, I could not run the scripts.

Reply:
Is it possible to do it in another PR?

Reply:
Reverted.